corehq/elastic.py

```diff
@@ -536,3 +540,6 @@ def report_shard_failures(search_result):
 
     if search_result.get('_shards', {}).get('failed'):
         datadog_counter('commcare.es.partial_results', value=1)
+        # Example message:
+        #   "_shards: {'successful': 2, 'failed': 0, 'total': 2}"
```

Comment:
I think this function (and docstring) is no longer well named as it now raises an exception.
Would it be useful to also include the query in the error message so we can figure out what happened? Or is that included in the stacktrace already?

Reply:
I think ES requests show up in the sentry breadcrumbs. I've confirmed that's the case for some ESError 500 sentry events, so I'm fairly sure about this